src/test/java/com/philips/research/spdxbuilder/core/domain/ConversionInteractorTest.java

```diff
@@ -119,18 +120,22 @@ void setsDocumentIdentification() {
     }
 
     @Test
-    void verifyPackageWithNoSupplierField() {
+    void verifySBOMFields() throws IOException {
         bom.addPackage(pkg);
+        bom.setCreatedAt("2021-02-15T09:47:02.105Z");
```

Comment:
Pass in a SimpleDate object here to leverage Java date parsing to prevent errorness inputs.